src/Screens/HomeScreen/HomeScreen.tsx

```diff
@@ -1,18 +1,25 @@
-import React from "react";
```

Comment:
Your PR title is incorrect. You should keep the name correct with the requirements of the project, and add simple description